DQM/Integration/python/clients/beamfake_dqm_sourceclient-live_cfg.py

```diff
@@ -57,12 +52,7 @@
 process.load("DQM.Integration.config.environment_cfi")
 process.dqmEnv.subSystemFolder = 'FakeBeamMonitor'
 process.dqmSaver.tag           = 'FakeBeamMonitor'
-process.dqmSaver.runNumber     = options.runNumber
-process.dqmSaverPB.tag         = 'FakeBeamMonitor'
-process.dqmSaverPB.runNumber   = options.runNumber
```

Comment:
Hello, is there a reason to remove the support of the `dqmSaverPB`? Also setting `process.dqmSaver.runNumber` is now required to make sure the client works correctly in our integration playback system.

Reply:
Hi,
so to the best of my knowledge dqmSaverPB was not used at all for the clients (even in the others I do not find trace of it. Is it important to add it back? Where I can find the configuration for that module?

Reply:
Hi Simone, `dqmSaverPB` is a recent addition that allows all DQM Online clients to communicate with the new version of the DQM GUI. The code for this is already in the master but I never fully backported it to 11_1. I just reopened the pull request #31206 that will be merged and that can be used by you in local testing.